test/src/unit-capi-dense_array.cc

```diff
@@ -5140,60 +5140,3 @@ TEST_CASE_METHOD(
 
   remove_temp_dir(temp_dir);
 }
-
-TEST_CASE_METHOD(
```

Comment:
I think this test case would pass. But it doesn't really matter for the release branch.